### PR TITLE
prov/rxd: set attr->op_flags correctly

### PR DESCRIPTION
**Pull request by aingerson**

### CODE REVIEW

**Comment by aingerson** [comment]
@a-ilango

File: prov/rxd/src/rxd_attr.c [hunk 1]
```diff
@@ -40,6 +40,7 @@
 
 struct fi_tx_attr rxd_tx_attr = {
 	.caps = RXD_EP_CAPS | RXD_TX_CAPS,
+	.op_flags = ~0ULL & ~FI_MULTICAST,
```
Comment (a-ilango) [inline prov/rxd/src/rxd_attr.c]: I think the op_flags should be set in a way similar to the changes in rxm in #5027

Comment (aingerson) [reply prov/rxd/src/rxd_attr.c]: Updated

**Comment by shefty** [comment]
It looks like @a-ilango pulled this commit into #5027 to merge with his set of changes.

**Comment by a-ilango** [comment]
Yes, I wanted to make sure it plays well with the util code fixes. The rxd tests that were failing before for #5027 are passing now.

**Pull request closed without merging**